STM32F405RGTx/Core/Inc/threads.hpp

```diff
@@ -6,12 +6,35 @@
 #include "state_machine.hpp"
 #include "ltc6813_thread.hpp"
 #include "bist_thread.hpp"
-#include "debug_led_thread.hpp"
-
-// In milliseconds
-#define MEASUREMENT_PERIODICITY         100
-#define LTC_MEASUREMENT_PERIODICITY     500
-#define BIST_PERIODICITY                50
-#define COULOMB_COUNTING_PERIODICITY    150
-#define DEBUG_LED_PERIODICITY           1000
-#define STATE_MACHINE_PERIODICITY       200
+#include "LED_thread.hpp"
+#include "CAN_thread.hpp"
+
+// see ./docs/utilization.xlsx for schedulability calculations
```

Comment:
Personally, I'm not a fan of a file not human readable on github. This is fine but do you think we could instead do this with python and out put a txt, md or json file from the script?. Excel is fine though if you think its just a lot simpler

Reply:
or maybe leave it as a csv? so that it's viewable in excel too

Reply:
There are formulas tho, CSVs can't have those